### PR TITLE
[Keyboard] Add Plaque80

### PR DESCRIPTION
**Pull request by MaiTheSan**
## Description
Add QMK support for PLAQUE PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sawnsprojects/plaque80/info.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        "audio": false,
+        "key_lock": false
+    },
+    "rgblight": {
+        "pin": "F0",
+        "led_count": 0,
+        "hue_steps": 8,
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "max_brightness": 255,
```
Comment (drashna) [inline keyboards/sawnsprojects/plaque80/info.json:36]:
```suggestion
```

File: keyboards/sawnsprojects/plaque80/info.json [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        "sleep": true,
+        "animations": {
+            "all": true
```
Comment (drashna) [inline keyboards/sawnsprojects/plaque80/info.json:39]:
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/sawnsprojects/plaque80/info.json [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        "command": false,
+        "nkro": false,
+        "backlight": false,
+        "rgblight": true,
+        "audio": false,
+        "key_lock": false
+    },
+    "rgblight": {
+        "pin": "F0",
+        "led_count": 0,
```
Comment (zvecr) [inline keyboards/sawnsprojects/plaque80/info.json:32]: Zero LEDs, either the rgblight config should be removed or this value corrected.

File: keyboards/sawnsprojects/plaque80/info.json [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": false,
+        "backlight": false,
+        "rgblight": true,
+        "audio": false,
+        "key_lock": false
```
Comment (zvecr) [inline keyboards/sawnsprojects/plaque80/info.json:28]:
```suggestion
        "rgblight": true
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**